python/pydevSrc/com/jetbrains/python/debugger/pydev/ProcessDebugger.java

```diff
@@ -89,4 +89,7 @@ void setBreakpointWithFuncName(@NotNull String typeId, @NotNull String file, int
   void addExceptionBreakpoint(ExceptionBreakpointCommandFactory factory);
 
   void removeExceptionBreakpoint(ExceptionBreakpointCommandFactory factory);
+
+  ArrayChunk loadPandasDataFrame(String id, String id1, PyDebugValue var, int offset, int offset1, int rows, int cols, String format)
```

Comment:
id1 and offset1 aren't good examples of variable naming. But as this method signature is the same as loadArrayItems we could reuse it having the separation only on python side based on the type of the variable.